doc/nrf/known_issues.rst

```diff
@@ -885,6 +885,19 @@ KRKNWK-6073: Potential delay during FOTA
 Matter
 ======
 
+.. rst-class:: v2-1-2 v2-1-1
+
+KRKNWK-15749: Invalid ZAP Tool revision used
+  The ZAP Tool revision used for the |NCS| v2.1.1 and v2.1.2 releases is not compatible with the :file:`zap` files that define the Matter Data Model.
```

Comment:
Mention that the incompatibility concerns only ZAP files from Matter samples in NCS? It's no that some generic Matter Data Model files are wrong.